homeassistant/components/camera/__init__.py

```diff
@@ -85,13 +85,21 @@
 from .prefs import CameraPreferences, DynamicStreamSettings  # noqa: F401
 from .webrtc import (
     DATA_ICE_SERVERS,
+    CameraWebRTCLegacyProvider,
     CameraWebRTCProvider,
     RTCIceServer,
+    WebRTCAnswer,
+    WebRTCCandidate,  # noqa: F401
```

Comment:
Side note: We can define an `__all__` variable to avoid needing to disable the linter. That will also make it clear what we want to export. We can address this in a follow up.